lib/plugins/issues.js

```diff
@@ -54,4 +54,13 @@ module.exports = class Issues extends Plugin {
 
     return deleteFunction(context.toRepo({id: comment.id}));
   }
+
+  create(context, content) {
```

Comment:
Since everything shares a namespace right now, I wonder if it would make sense to call this `createIssue` to disambiguate between other create actions like `createPullRequest`, `createRelease`, etc…